egov-persister/judge-calendar-persister.yml

```
@@ -11,37 +11,28 @@ serviceMaps:
             - jsonPath: $.JudgeCalendarRule.*.notes
-
             - jsonPath: $.JudgeCalendarRule.*.auditDetails.lastModifiedBy
-
             - jsonPath: $.JudgeCalendarRule.*.auditDetails.lastModifiedTime
-
-            - jsonPath: $.JudgeCalendarRule.*.tenantId
\ No newline at end of file
+            - jsonPath: $.JudgeCalendarRule.*.tenantId
+            - jsonPath: $.JudgeCalendarRule.*.courtIds
+              type: JSON
+              dbType: JSONB
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)